jbehave-support-core/src/main/java/org/jbehavesupport/core/web/WebTableSteps.java

```diff
@@ -30,7 +30,7 @@
 import org.springframework.stereotype.Component;
 
 /**
- * @deprecated support for general usable table steps will be added in future version and it is not guarantee that it will be backward compatible
+ * @deprecated(since = "1.0.0", forRemoval = true) support for general usable table steps may be added in a future version and it is not guaranteed that it will be backwards compatible
  */
```

Comment:
Are those steps really 'forRemoval=true'?

Reply:
Based on the current wording and parameters I presume we will eventually remove the steps alltogether, but definitely not before we have some suitable replacement in terms of functionality. So for removal makes sense, but not until we have some replacement which we dont have and dont know what it should be yet (should have been solved by #60 )